2023-07-06 23:16:52,887 INFO    MainThread:18867 [wandb_setup.py:_flush():76] Current SDK version is 0.15.4
2023-07-06 23:16:52,887 INFO    MainThread:18867 [wandb_setup.py:_flush():76] Configure stats pid to 18867
2023-07-06 23:16:52,887 INFO    MainThread:18867 [wandb_setup.py:_flush():76] Loading settings from /home/ubuntu/.config/wandb/settings
2023-07-06 23:16:52,887 INFO    MainThread:18867 [wandb_setup.py:_flush():76] Loading settings from /home/ubuntu/Documents/working/Hangers.AI/DemandFocasting/gtm_implement/wandb/settings
2023-07-06 23:16:52,887 INFO    MainThread:18867 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2023-07-06 23:16:52,887 INFO    MainThread:18867 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2023-07-06 23:16:52,887 INFO    MainThread:18867 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program_relpath': 'train.py', 'program': 'train.py'}
2023-07-06 23:16:52,887 INFO    MainThread:18867 [wandb_init.py:_log_setup():507] Logging user logs to /home/ubuntu/Documents/working/Hangers.AI/DemandFocasting/gtm_implement/wandb/run-20230706_231652-vpqz2okp/logs/debug.log
2023-07-06 23:16:52,887 INFO    MainThread:18867 [wandb_init.py:_log_setup():508] Logging internal logs to /home/ubuntu/Documents/working/Hangers.AI/DemandFocasting/gtm_implement/wandb/run-20230706_231652-vpqz2okp/logs/debug-internal.log
2023-07-06 23:16:52,887 INFO    MainThread:18867 [wandb_init.py:init():547] calling init triggers
2023-07-06 23:16:52,887 INFO    MainThread:18867 [wandb_init.py:init():555] wandb.init called with sweep_config: {}
config: {}
2023-07-06 23:16:52,888 INFO    MainThread:18867 [wandb_init.py:init():596] starting backend
2023-07-06 23:16:52,888 INFO    MainThread:18867 [wandb_init.py:init():600] setting up manager
2023-07-06 23:16:52,893 INFO    MainThread:18867 [backend.py:_multiprocessing_setup():108] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2023-07-06 23:16:52,900 INFO    MainThread:18867 [wandb_init.py:init():606] backend started and connected
2023-07-06 23:16:52,909 INFO    MainThread:18867 [wandb_init.py:init():703] updated telemetry
2023-07-06 23:16:52,910 INFO    MainThread:18867 [wandb_init.py:init():736] communicating run to backend with 60.0 second timeout
2023-07-06 23:16:53,440 ERROR   MainThread:18867 [wandb_init.py:init():762] encountered error: It appears that you do not have permission to access the requested resource. Please reach out to the project owner to grant you access. If you have the correct permissions, verify that there are no issues with your networking setup.(Error 403: Forbidden)
2023-07-06 23:16:53,695 ERROR   MainThread:18867 [wandb_init.py:init():1170] It appears that you do not have permission to access the requested resource. Please reach out to the project owner to grant you access. If you have the correct permissions, verify that there are no issues with your networking setup.(Error 403: Forbidden)
Traceback (most recent call last):
  File "/home/ubuntu/Documents/working/Hangers.AI/.venv/lib/python3.7/site-packages/wandb/sdk/wandb_init.py", line 1152, in init
    run = wi.init()
  File "/home/ubuntu/Documents/working/Hangers.AI/.venv/lib/python3.7/site-packages/wandb/sdk/wandb_init.py", line 768, in init
    raise error
wandb.errors.CommError: It appears that you do not have permission to access the requested resource. Please reach out to the project owner to grant you access. If you have the correct permissions, verify that there are no issues with your networking setup.(Error 403: Forbidden)
2023-07-06 23:16:54,697 WARNING MsgRouterThr:18867 [router.py:message_loop():77] message_loop has been closed
